cmd/create/cluster/cmd.go

```diff
@@ -63,8 +67,11 @@ var args struct {
 	version            string
 	channelGroup       string
 	flavour            string
-	etcdEncryption     bool
 
+	//Encryption
+	etcdEncryption           bool
+	enableCustomerManagedKey bool
```

Comment:
What's the purpose of this flag, is it just to gate the kms key? If so, we can probably remove it, as the logic to determine if the key arn is needed can be done by just verifying that there's a value in it. Basically, if the kmsKeyARN is set, it's used, if it's empty, it's not.

Reply:
@vkareh I was thinking of mimicking what our portal has, and it will also be more informative for the user to understand what they are really doing when they provide kms key arn. They have turned on custom encryption for their EBS volume. If you believe it is unnecessary, I can remove it. Please confirm

Reply:
Okay, this is a good point. In this case I have a suggestion: if the user sets `--kms-key-arn`, then skip the check and assume that `enable-customer-managed-key` is implied true.

So I think we need to ensure the following behaviours:
- if user sets only `--interactive` we show both prompts
- if user sets `--enable-customer-managed-key` without the `--kms-key-arn` we force interactive mode and prompt for the kms key arn
- if user sets `--kms-key-arn` (with or without the `--enable-customer-managed-key`) we ignore enableCustomerManagedKey, since it's implied true

Does that make sense?